src/app/portlets/dot-templates/dot-template-designer/dot-template-designer.component.ts

```diff
@@ -20,10 +20,10 @@ export class DotTemplateDesignerComponent implements OnInit {
     form: FormGroup;
```

Comment:
You should not be change it anything in this file, it could lead to issues in my merge.